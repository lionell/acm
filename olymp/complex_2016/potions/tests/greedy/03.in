2000 9
57 22
31 84
20 46
8 15
36 87
4 6
46 52
35 22
15 20
17 76
81 88
67 58
51 64
56 19
8 86
6 21
3 62
8 36
48 20
8 13
17 84
36 9
30 62
81 69
45 42
31 95
54 26
87 97
16 3
86 42
60 21
82 89
25 63
80 88
98 70
54 87
93 15
76 88
55 24
65 58
53 99
23 95
92 5
5 9
50 6
24 90
79 10
70 5
92 2
72 49
36 90
4 24
10 30
51 79
35 64
61 16
10 86
90 84
92 99
4 93
83 40
47 27
31 61
60 18
66 21
10 30
54 99
27 13
91 61
26 75
89 27
13 86
68 98
95 1
93 60
99 98
26 75
36 47
62 56
77 93
24 26
23 84
96 77
36 48
23 87
12 61
46 12
8 22
21 14
72 4
2 12
86 69
14 25
81 20
14 73
98 56
40 37
12 20
65 34
21 47
7 87
97 31
50 52
63 3
6 70
82 35
3 6
30 66
86 16
89 9
62 97
34 85
3 2
33 43
89 67
55 52
82 95
45 50
52 30
1 9
41 56
59 80
44 43
59 86
95 28
25 46
29 57
57 56
98 32
98 89
41 86
79 51
1 21
51 25
31 51
5 49
27 71
13 63
49 70
98 69
15 42
97 21
77 43
72 52
39 75
59 71
20 78
96 39
63 21
69 45
91 91
62 73
34 18
86 72
41 80
23 82
3 53
94 18
69 77
51 66
35 8
83 10
48 55
73 80
25 9
99 42
13 17
32 60
30 44
24 18
97 71
24 44
61 98
75 18
82 28
36 24
34 17
71 17
96 79
86 44
83 22
36 83
41 93
37 67
83 71
42 59
3 80
78 63
79 62
89 52
73 61
77 23
40 7
84 49
90 36
55 70
51 74
67 90
56 92
61 3
61 39
17 3
63 61
23 93
43 42
3 13
35 16
21 80
27 72
9 5
72 17
88 61
50 21
13 54
54 6
42 73
73 15
73 56
49 35
76 94
7 91
6 77
55 39
12 26
28 81
95 18
78 99
20 82
36 26
31 31
2 87
2 70
27 73
8 72
66 73
63 83
58 70
8 68
93 13
91 17
34 21
18 85
67 10
36 35
63 1
87 64
33 65
38 86
58 59
32 43
26 23
92 92
60 82
94 99
16 51
69 86
71 47
55 84
18 36
36 88
52 80
43 21
7 84
41 80
23 62
84 71
63 47
29 75
74 22
72 21
5 88
33 40
24 73
9 87
75 41
22 42
62 26
8 64
43 66
29 49
20 63
8 11
85 82
4 34
55 58
46 74
14 59
31 76
62 37
76 39
79 35
60 97
59 39
6 68
17 99
63 32
42 34
14 68
3 28
83 16
89 57
14 27
3 1
35 42
81 64
99 11
6 58
96 57
23 63
62 99
31 37
70 23
91 71
98 84
97 93
48 79
6 84
84 59
2 8
70 19
28 80
38 67
24 33
93 32
32 47
83 56
78 60
31 52
34 67
58 27
6 31
15 6
63 11
19 98
15 62
42 88
53 40
71 77
7 76
21 64
20 38
98 3
52 96
63 29
56 85
16 22
25 59
70 28
24 85
47 88
74 36
76 86
61 27
1 47
11 68
4 20
20 31
25 99
27 71
54 87
7 80
39 67
94 30
15 9
94 18
53 60
44 66
90 27
73 6
71 91
9 84
13 74
72 26
96 37
22 98
78 50
17 27
56 17
24 9
24 68
28 18
84 74
2 70
75 74
63 74
56 44
18 69
94 66
3 87
85 88
38 23
49 63
78 52
58 3
70 99
17 81
56 96
66 98
71 57
29 39
82 34
2 85
51 97
85 95
84 52
72 70
32 20
70 19
21 8
7 28
8 88
82 21
20 63
18 48
84 88
22 46
29 64
62 21
17 78
28 45
15 98
18 99
18 44
49 85
12 36
22 53
74 19
79 5
50 91
79 96
40 35
98 98
20 67
45 58
3 36
34 70
69 16
57 49
35 84
18 7
59 44
62 39
41 31
22 41
35 91
24 1
97 72
39 20
78 17
52 81
51 78
91 83
30 18
99 49
53 62
6 17
54 10
41 67
6 94
85 60
59 40
13 9
27 56
72 49
31 3
80 22
3 81
99 70
17 32
94 98
14 69
76 97
65 67
61 15
75 68
9 45
51 32
87 19
67 77
79 57
78 95
77 58
28 80
10 74
72 44
11 2
99 83
50 84
99 62
30 10
54 72
4 36
52 5
80 88
45 20
15 57
15 21
98 89
63 40
81 8
8 34
17 91
76 7
68 65
74 75
45 95
30 26
30 48
34 81
98 8
65 78
98 12
1 77
18 96
2 62
95 98
88 9
13 10
74 64
37 78
74 46
71 79
27 4
84 99
7 58
36 81
92 69
47 32
26 92
52 62
58 27
34 47
55 46
8 46
23 27
73 42
21 96
97 44
43 45
3 79
60 48
15 36
67 52
43 59
21 90
16 94
40 77
21 49
92 94
19 27
68 15
9 91
34 87
30 6
83 74
20 31
64 43
93 35
93 30
20 36
30 13
89 35
81 69
61 10
34 74
86 78
69 1
87 95
98 1
73 15
43 81
23 92
25 8
35 16
51 19
29 54
89 78
46 18
25 70
42 5
82 59
59 29
22 50
50 44
56 20
99 21
14 97
3 22
37 38
54 37
90 85
61 82
97 77
48 7
12 21
77 87
16 91
41 34
77 35
55 89
10 31
28 54
73 21
57 30
66 8
90 11
91 55
32 51
58 88
7 77
65 67
58 83
17 78
14 97
86 91
23 66
18 95
17 48
78 90
97 73
81 42
94 87
36 72
59 27
4 93
58 65
49 66
43 17
14 65
55 56
22 97
92 77
25 39
27 7
77 1
41 22
7 57
27 35
59 42
34 85
49 63
27 91
6 95
58 69
23 17
14 13
89 44
80 5
9 13
11 5
26 86
41 51
84 32
74 67
51 43
4 6
96 99
94 31
97 3
19 52
65 21
62 31
35 52
62 41
45 43
27 73
22 71
1 67
33 6
47 72
77 83
82 50
78 72
74 74
26 76
94 90
21 88
39 54
94 53
96 1
73 38
7 21
86 93
98 7
77 19
2 43
93 54
26 81
55 69
43 97
88 78
65 37
89 7
58 4
4 83
21 54
72 75
68 27
32 58
76 64
8 9
62 77
56 98
67 86
83 9
85 9
45 69
74 50
51 34
16 32
83 52
25 34
59 55
14 92
55 90
98 89
65 62
58 58
42 19
28 23
31 24
90 13
60 74
6 64
93 9
58 21
55 76
30 83
73 14
2 43
30 28
87 99
55 92
12 45
68 97
21 37
49 98
70 11
73 7
13 76
96 64
41 70
51 49
60 68
11 25
52 61
58 38
31 38
80 12
7 40
76 46
43 25
33 23
27 13
88 6
70 39
7 85
32 9
76 58
82 92
51 84
20 32
67 8
19 48
87 48
93 23
47 61
83 36
49 79
83 10
46 37
20 51
59 53
11 52
42 33
15 90
20 90
97 32
80 87
35 14
36 65
27 28
62 80
60 10
18 11
47 43
91 63
14 65
17 48
78 22
12 56
46 92
22 32
17 42
55 2
65 51
76 89
69 91
98 38
48 27
69 16
77 93
58 58
6 90
13 72
28 83
73 22
51 71
13 95
94 67
16 66
55 59
48 92
30 24
51 47
60 76
69 18
74 37
26 25
94 79
63 36
55 20
90 34
29 7
71 2
67 24
80 87
79 22
43 29
75 9
82 91
8 33
70 51
73 81
60 93
27 67
84 21
55 81
87 75
76 81
3 59
44 43
62 83
12 23
29 5
96 84
17 12
60 2
82 84
75 32
96 42
61 2
83 81
54 16
96 68
25 30
70 97
80 68
88 30
34 89
71 18
27 28
29 85
68 84
16 11
51 43
43 13
93 12
25 24
89 45
74 19
16 13
80 42
71 95
82 67
82 3
29 53
37 9
90 55
66 5
45 7
19 15
24 85
9 10
54 48
66 96
9 26
65 80
73 86
53 34
36 56
7 35
41 62
18 41
45 32
36 83
95 88
73 52
59 19
64 80
74 11
34 28
8 82
68 98
32 81
35 20
52 65
28 39
80 91
21 45
26 23
11 56
8 19
37 82
60 66
74 98
26 34
14 8
6 32
11 81
98 35
97 43
81 51
42 25
67 59
82 62
16 90
9 90
72 21
86 43
41 30
61 58
65 65
94 74
54 68
3 3
45 52
3 97
21 26
85 42
2 87
76 65
55 15
35 84
28 26
53 19
76 66
29 15
88 41
9 21
23 42
94 9
7 67
91 94
35 27
12 74
37 34
49 87
72 91
17 83
3 97
61 68
82 78
18 88
9 70
11 24
33 32
96 3
96 37
63 85
59 29
63 72
60 95
85 10
92 30
25 99
67 92
69 85
73 49
18 86
10 82
12 28
30 40
76 8
93 24
53 39
10 50
44 14
23 69
96 29
28 15
6 21
6 95
42 74
60 77
59 59
86 67
8 68
75 14
38 83
21 66
17 88
2 30
96 60
88 24
36 92
13 14
7 41
13 16
92 47
6 70
38 49
17 90
4 43
24 90
56 39
27 44
71 72
30 26
47 65
55 16
30 81
20 67
81 34
80 33
3 71
21 84
74 38
80 37
27 75
13 5
46 81
53 37
63 16
80 81
96 8
88 36
1 24
56 9
39 79
51 36
18 41
79 71
6 89
65 59
61 31
10 75
12 8
23 60
41 72
80 1
34 36
59 68
74 33
10 15
49 13
54 60
29 66
56 31
87 33
61 19
91 49
54 70
30 3
75 74
72 71
7 55
21 5
37 63
76 95
8 46
5 25
88 59
89 33
63 42
58 75
22 24
93 49
52 73
47 24
92 25
79 17
21 98
61 99
92 55
99 37
59 1
59 4
34 46
87 47
20 97
21 46
92 41
12 13
36 42
66 58
72 26
24 43
41 93
48 82
18 33
33 47
48 76
20 90
35 82
77 8
51 54
92 97
8 43
85 4
59 41
66 49
92 29
22 89
70 33
63 68
13 87
61 94
84 61
41 81
86 17
68 18
13 37
77 58
59 21
61 60
10 18
44 28
15 99
32 66
32 84
71 92
86 44
3 31
12 70
84 43
60 97
32 52
10 73
93 7
67 68
83 52
79 74
73 28
91 94
78 3
94 23
64 48
78 81
49 67
10 87
82 34
83 68
41 14
20 93
59 32
83 84
58 43
68 63
55 29
31 59
79 31
78 24
5 43
8 54
41 51
82 18
83 21
32 66
59 24
53 50
34 18
58 36
96 90
19 26
84 49
80 48
69 61
3 58
11 73
22 10
25 49
70 5
70 8
31 2
51 29
44 81
18 83
73 3
26 12
61 89
35 9
69 41
96 4
74 71
81 7
56 96
1 4
12 24
25 68
94 40
22 74
57 38
38 37
48 30
17 64
70 9
47 52
55 39
9 44
48 30
24 87
90 2
25 22
89 99
40 48
22 83
19 59
95 76
4 55
17 43
50 20
71 86
23 96
38 25
52 31
18 86
85 75
96 9
8 11
56 85
66 45
4 77
51 82
37 96
38 54
73 53
37 87
81 43
67 60
90 20
4 17
91 7
13 88
96 87
70 20
63 52
27 34
16 64
61 75
29 53
4 98
83 3
43 41
1 64
81 8
24 88
94 82
69 15
99 7
25 65
16 67
1 85
49 40
16 16
66 10
6 42
44 70
9 88
53 84
91 7
92 32
15 15
27 87
91 83
46 26
92 14
98 59
97 90
5 48
55 13
54 70
38 61
49 96
80 44
50 2
32 70
84 41
28 46
29 12
35 17
31 72
32 26
16 29
74 29
39 20
88 30
90 90
84 24
68 39
39 63
33 16
56 70
14 15
26 81
98 40
12 59
84 27
55 41
67 97
18 28
57 4
92 4
27 47
83 76
37 92
7 20
87 69
82 62
42 1
40 8
66 38
64 50
90 48
46 17
44 56
59 61
64 1
45 49
24 88
79 27
46 60
28 85
47 32
32 10
15 86
24 70
20 81
27 85
3 8
61 72
33 44
43 19
67 96
83 87
12 90
51 62
47 56
87 78
85 91
78 17
86 99
78 99
85 4
10 6
77 85
29 70
88 11
5 71
57 56
46 85
47 68
24 94
70 93
83 9
26 55
53 59
58 10
14 31
34 43
26 23
90 12
20 55
24 79
33 22
8 78
47 76
71 54
46 68
75 39
94 29
87 1
8 45
75 43
86 21
41 10
19 10
64 32
9 36
57 86
64 39
15 64
19 9
75 83
23 62
90 50
48 15
59 75
18 55
76 32
41 2
12 15
44 57
92 73
57 50
88 50
14 22
30 2
84 30
89 3
52 7
21 79
52 99
55 79
12 68
69 29
41 50
6 78
50 83
33 97
45 7
26 22
23 57
84 54
56 5
12 74
51 9
6 30
8 4
71 58
84 17
64 38
15 26
9 69
65 62
68 41
60 10
64 92
46 80
85 49
23 2
8 94
24 73
74 14
69 31
45 43
79 54
79 10
78 93
53 85
24 43
51 21
11 82
62 15
63 56
56 45
39 85
57 61
72 61
89 31
74 42
95 35
44 53
44 72
55 22
63 96
15 78
60 14
26 26
79 21
63 89
72 33
94 3
61 27
58 66
6 50
84 30
80 1
70 26
46 24
18 25
2 9
20 32
55 61
79 45
32 33
65 43
43 3
29 57
24 4
51 84
14 27
28 36
61 93
15 95
21 7
13 32
62 20
80 30
73 15
45 58
98 3
5 11
65 41
42 34
18 86
14 92
28 30
21 39
34 87
91 36
65 52
71 3
32 25
38 50
8 5
8 82
90 6
44 10
41 56
42 85
77 59
86 55
93 4
90 7
42 28
77 79
81 6
30 49
98 12
14 67
48 4
7 19
29 38
93 50
35 69
26 36
90 10
13 10
17 82
8 1
80 56
61 84
33 59
68 89
54 29
30 82
1 99
38 36
83 27
94 31
64 18
28 18
28 55
35 38
39 44
97 42
26 17
73 57
46 57
83 42
24 97
97 12
47 22
47 33
63 31
46 41
58 28
80 71
10 85
27 15
54 46
62 25
81 77
34 35
74 25
20 16
27 95
18 15
47 71
2 62
1 11
36 46
17 58
42 14
26 24
69 66
88 79
56 29
62 69
91 88
2 36
32 10
24 28
98 47
7 71
79 98
42 7
62 13
26 56
80 2
68 51
28 47
74 56
24 83
69 36
69 15
22 70
97 98
43 46
17 94
90 49
53 93
6 32
85 16
17 29
79 63
11 84
39 7
87 82
16 62
74 57
25 84
81 96
40 21
15 24
71 54
47 3
32 24
39 50
78 17
80 54
36 55
59 88
70 74
34 46
1 85
67 7
3 26
46 48
70 42
93 58
58 41
64 40
89 89
6 2
53 65
20 83
69 88
61 78
22 39
22 94
1 23
46 89
35 1
40 89
47 5
45 33
70 5
91 8
7 57
22 94
78 59
45 41
19 47
84 6
98 39
61 6
92 96
95 5
93 28
32 35
66 38
40 75
82 36
93 31
23 86
43 15
54 1
46 85
91 70
9 28
31 87
83 67
71 24
51 76
12 62
99 80
55 75
11 37
65 35
19 2
15 88
86 61
45 68
36 30
57 36
21 45
12 88
10 2
78 81
43 60
38 87
62 41
77 91
24 70
72 41
29 43
2 86
52 15
43 46
79 87
30 99
85 97
99 40
21 94
52 75
62 61
2 90
79 22
91 77
18 3
43 63
47 46
58 43
86 96
84 1
98 64
61 14
51 83
78 58
33 71
33 28
16 92
14 32
7 94
95 4
64 24
67 38
78 12
8 24
22 65
29 89
2 20
2 89
46 52
22 77
5 78
69 52
83 21
13 80
83 88
12 8
44 48
57 77
1 22
86 62
52 22
40 13
3 51
1 41
18 46
22 22
73 20
40 90
69 55
43 53
58 52
97 52
29 2
24 52
15 27
49 8
21 64
15 86
27 21
66 15
35 42
61 87
76 6
60 2
52 45
94 1
53 10
12 90
42 79
7 33
40 54
18 53
39 60
78 30
43 63
5 44
30 77
82 65
66 6
48 41
39 16
26 40
30 89
69 35
67 70
24 73
26 7
65 40
69 62
25 43
85 10
84 29
93 15
18 65
6 58
71 65
5 44
34 96
29 33
3 1
73 95
1 24
63 96
56 63
4 31
40 78
7 86
1 23
87 97
53 16
78 92
34 25
19 81
14 65
63 48
41 14
37 36
30 39
99 1
31 83
61 4
89 68
90 65
61 88
4 77
67 14
36 80
61 99
62 55
1 74
85 26
59 39
78 22
22 87
70 75
78 50
17 30
94 66
52 5
82 55
67 53
33 47
44 3
57 91
63 7
30 55
94 48
70 88
75 70
45 89
40 44
73 22
85 54
58 65
21 37
87 38
85 85
87 21
12 27
31 43
97 73
21 60
47 89
59 90
80 20
63 5
26 18
71 34
99 11
45 29
67 18
3 32
50 50
74 87
30 60
33 6
63 26
15 53
44 10
30 74
76 22
37 92
25 99
8 6
32 24
39 52
83 96
46 41
28 32
91 19
22 56
81 24
76 84
94 95
67 18
37 22
12 41
40 73
78 36
57 47
96 11
5 95
36 78
10 49
67 62
18 1
24 88
72 98
91 99
16 64
83 57
97 52
26 95
29 38
82 2
10 86
81 79
55 15
63 16
77 64
64 30
16 94
90 85
82 85
48 81
38 97
50 31
24 34
69 75
74 53
39 51
28 84
98 20
35 80
44 59
86 10
4 6
88 2
84 91
73 71
68 32
61 9
40 18
90 84
37 9
58 65
47 75
93 83
64 43
2 26
36 6
12 87
86 37
29 97
68 68
1 99
6 37
52 61
43 46
52 41
4 80
53 8
91 50
92 46
70 53
58 93
78 6
40 67
65 62
30 66
63 33
67 28
86 69
13 18
58 30
10 62
70 61
11 62
7 61
12 1
93 76
79 69
37 69
