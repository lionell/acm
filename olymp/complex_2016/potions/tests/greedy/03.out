0.0129603